app/Http/Livewire/SlackSettingsForm.php

```diff
@@ -174,8 +184,40 @@ public function submit()
             $this->setting->save();
 
             session()->flash('success',trans('admin/settings/message.update.success'));
-
         }
 
     }
+     public function msTeamTestWebhook(){
+
+     $payload =
+        [
+            "@type" => "MessageCard",
+            "@context" => "http://schema.org/extensions",
+            "summary" => "Snipe-IT Integration Test Summary",
```

Comment:
We should probably make these into translations

Reply:
Github's view for this made it seem like you meant `@type` and `@context` but you really mean `summary` and `title` right?

Reply:
Yes